src/App.jsx

```diff
@@ -14,70 +16,77 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const indexOfItem = goods.indexOf('Jam');
```

Comment:
the name of the variable does not clearly indicate what it contains
better rename to `defaultGoodIndex`, `indexOfJam`, `goodJamIndex`